optimizely/oui
# EditableInput component

**Summary**

Adding an `EditableTextField` component, that has similar behavior to the html5 `contenteditable` attribute. 


![editabletextfield-demo](https://user-images.githubusercontent.com/41028823/43809774-c31c9564-9a79-11e8-9749-2e294dba2ed7.gif)





**Test Plan**

1. Run the test suite `npm test`
2. Verify the snapshots are correct
3. Run storybook and click around

**Jira**
no ticket
